app/products/calls/connection/connection.ts

```
@@ -103,13 +103,45 @@ export async function newConnection(
 
     InCallManager.start();
     InCallManager.stopProximitySensor();
 
-    // Throws an error, to be caught by caller.
-    await ws.initialize();
+    try {
+        await ws.initialize();
+    } catch (err) {
+        InCallManager.stop();
+        audioDeviceChanged?.remove();
+        // Rethrows the error, to be caught by the caller.
```

ESLint

[error] Expected line before comment.

(lines-around-comment)